src/interfaces/modules/auctions/IEncryptedDerivativeValue.sol

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity >=0.8.0;
+
+import {Point} from "src/lib/ECIES.sol";
+
+/// @title  IEncryptedDerivativeValue
+/// @notice Interface for encrypted derivative value batch auctions
+/// @dev    This contract does not inherit from `BatchAuctionModule` in order to avoid conflicts. Implementing contracts should inherit from both `BatchAuctionModule` and this interface.
+interface IEncryptedDerivativeValue {
+    // ========== ERRORS ========== //
+
+    error Auction_InvalidKey();
+    error Auction_WrongState(uint96 lotId);
+    error Bid_WrongState(uint96 lotId, uint64 bidId);
+    error NotPermitted(address caller);
+
+    // ========== EVENTS ========== //
+
+    event BidDecrypted(uint96 indexed lotId, uint64 indexed bidId, uint128 value);
+
+    // ========== DATA STRUCTURES ========== //
+
+    /// @notice         Parameters that are used to set auction-specific data
+    ///
+    /// @param          price               The fixed price of the auction (quote tokens per base tokens)
+    /// @param          minValue            The minimum derivative value that a user must bid to be considered in the auction
+    /// @param          sortHighToLow       True if bid values are sorted high to low, false if low to high
```

Comment:
There may be instances where you want the winners to be whoever bids the lowest value for some parameter. Some examples are: the duration of an option, the size of an additional incentive, the strike price of a put option.

Because of this `minValue` may need to conditionally be `maxValue`